test/support/page_objects/dashboard_page.js

```diff
@@ -12,20 +15,77 @@ export default class DashboardPage {
     this.findTimeout = this.remote.setFindTimeout(defaultFindTimeout);
   }
 
-  gotoDashboardLandingPage() {
-    return this.findTimeout
-      .findByCssSelector('a[href="#/dashboard"]')
-      .click();
+  async initTests() {
+    const logstash = scenarioManager.loadIfEmpty('logstashFunctional');
+    await esClient.deleteAndUpdateConfigDoc({ 'dateFormat:tz':'UTC', 'defaultIndex':'logstash-*' });
+
+    PageObjects.common.debug('load kibana index with visualizations');
+    await elasticDump.elasticLoad('dashboard','.kibana');
+
+    await PageObjects.common.navigateToApp('dashboard');
+
+    return logstash;
+  }
+
+  /**
+   * Returns true if already on the dashboard landing page (that page doesn't have a link to itself).
+   * @returns {Promise<boolean>}
+   */
+  onDashboardLandingPage() {
+    PageObjects.common.debug(`onDashboardLandingPage`);
+    return PageObjects.common.findByCssSelector('a[href="#/dashboard"]', 1000)
+      .then(() => false)
+      .catch(() => true);
```

Comment:
I'm glad you pointed it out anyhow because as it turned out, it wasn't adhering to my override, I had to make some additional changes to get it to stick.